src/CoreWCF.NetTcp/src/CoreWCF/Dispatcher/ErrorBehaviorHelper.cs

```diff
@@ -32,7 +32,7 @@ internal static void ThrowAndCatch(Exception e, Message message)
                     }
                     else
                     {
-                        TraceUtility.ThrowHelperError(e, message);
+                        throw TraceUtility.ThrowHelperError(e, message);
```

Comment:
This was intentional. The idea is to throw if a debugger is attached, otherwise don't throw. Your change makes the if block and the else block identical.

Reply:
Ok, I see